### PR TITLE
add pdfobject v2.0.201604172 w/ npm autoupdate

### LINKED ISSUE #7897: [Request] Add PDFObject
**Library name:** PDFObject 
**Git repository url:** https://github.com/pipwerks/PDFObject
**npm package url(optional):** 
**License(s):** MIT-style license
**Official homepage:** https://pdfobject.com/
**Wanna say something? Leave message here:**  An open-source standards-friendly JavaScript utility for embedding PDF files into HTML documents.

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by nolsherry**
@dannyxx001 , I have fixed the wrong repo url, please help me to review this pull request, thanks!
Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/pipwerks/PDFObject
 * Official website (optional, not the repository): https://pdfobject.com/
 * NPM package url (optional): https://www.npmjs.com/package/pdfobject
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 437
   - Count of watchers: 48
   - Count of forks: 172
 * NPM download stats (optional):
   - Downloads in the last day: 7
   - Downloads in the last week: 81
   - Downloads in the last month: 331

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [X] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [X] The parent of the commit(s) in the PR is not old than 3 days.
 * [X] Pull request is sending from a non-master branch with meaningful name.
 * [X] Separate unrelated changes into different commits.
 * [X] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [X] Close corresponding issue in commit message
 * [X] Mention related issue(s), people in commit message, comment.

close #7897, cc @pipwerks

### CODE REVIEW

File: ajax/libs/pdfobject/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "name": "pdfobject",
+  "version": "2.0.201604172",
+  "description": "An open-source standards-friendly JavaScript utility for embedding PDF files into HTML documents",
+  "filename": "pdfobject.min.js",
+  "npmName": "pdfobject",
+  "npmFileMap": [
+    {
+      "basePath": "",
+      "files": [
+        "*.js"
```
Comment (pvnr0082t) [inline ajax/libs/pdfobject/package.json]: @nolsherry Let's use `pdfobject*` here.

File: ajax/libs/pdfobject/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "pdf",
+    "pdfobject"
+  ],
+  "author": {
+    "name": "Philip Hutchison",
+    "email": "philip@pipwerks.com",
+    "url": "http://pipwerks.com"
+  },
+  "license": "MIT",
+  "homepage": "https://github.com/pipwerks/PDFObject#readme"
```
Comment (pvnr0082t) [inline ajax/libs/pdfobject/package.json]: Let's use `https://pdfobject.com/` here.

**Comment by nolsherry** [comment]
@pvnr0082t, I have fixed the files format in npmFileMap and homepage url, please help me to review this pull request again, thanks.

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

File: ajax/libs/pdfobject/package.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    {
+      "basePath": "",
+      "files": [
+        "pdfobject*"
+      ]
+    }
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://pipwerks@github.com/pipwerks/PDFObject.git"
```
Comment (PeterDaveHello) [inline ajax/libs/pdfobject/package.json]: I don't think you have account of **pipwerks**, please remove this.

**Comment by nolsherry** [comment]
@pvnr0082t, I have fixed the wrong url in repo, please help me to review this pull request again, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Pull request merged**